src/v/debug_bundle/BUILD

```diff
@@ -0,0 +1,28 @@
+load("//bazel:build.bzl", "redpanda_cc_library")
+
+redpanda_cc_library(
+    name = "debug_bundle",
+    srcs = [
+        "debug_bundle_service.cc",
+        "error.cc",
+        "types.cc",
+    ],
+    hdrs = [
+        "debug_bundle_service.h",
+        "error.h",
+        "types.h",
+    ],
+    include_prefix = "debug_bundle",
+    visibility = ["//visibility:public"],
+    deps = [
+        "//src/v/base",
+        "//src/v/config",
+        "//src/v/container:fragmented_vector",
+        "//src/v/model",
+        "//src/v/strings:string_switch",
+        "//src/v/utils:uuid",
+        "@abseil-cpp//absl/container:btree",
+        "@fmt",
+        "@seastar",
+    ],
```

Comment:
nit: some of these can be `implementation_deps` if they aren't exposed in any headers. Looks like some of these can be removed (absl, frag_vec, etc).

Reply:
I think quite a few of these are exposed in `types.h`